zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/configuration/IdentityRequestCfg.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.impl.configuration;
+
+import java.util.Objects;
+
+public class IdentityRequestCfg {
```

Comment:
:upside_down_face: imo this is more of a `IdentityServiceCfg`, but honestly this is very much a nit :melting_face:

Reply:
Sounds better to me, I'll refactor it.